draft-ietf-tls-rfc8446bis.md

```diff
@@ -2932,7 +2933,7 @@ for each scenario:
 |------|-------------------|----------|
 | Server | ClientHello ... later of EncryptedExtensions/CertificateRequest | server_handshake_traffic_secret |
 | Client | ClientHello ... later of server Finished/EndOfEarlyData | client_handshake_traffic_secret |
-| Post-Handshake | ClientHello ... client Finished + CertificateRequest | client_application_traffic_secret_N |
+| Post-Handshake | ClientHello ... client Finished + CertificateRequest | [sender]_application_traffic_secret_N |
```

Comment:
```suggestion
| Post-Handshake | ClientHello ... client Finished + CertificateRequest | \[sender]_application_traffic_secret_N |
```

I'm not 100% on this, but I think that you need this.

Reply:
It seems to compile OK without this.

![image](https://github.com/user-attachments/assets/221ca031-2493-47e2-8341-785b70d0754b)